graphql/schema.graphql

```diff
@@ -161,6 +161,10 @@ type Mutation {
     @withSession
     @cacheControl(scope: PRIVATE)
     @withSender
+
+  ignoreB2BSessionData(enabled: Boolean!): MutationResponse
```

Comment:
Is the operation unauthenticated? If not, you should put `@checkUserAccess` to validate the user token access. We made the effort to add this directive to validate some operations.

Reply:
Yeah, the mutation only sets a public property in the user's session, so technically it doesn't matter if they're logged in or not, and I don't see any reason to check their access. If it's a non-B2B user it won't have any effect.

Reply:
Yeah, it looks like there is no down side in leaving it open to unauthenticated users. This would be a noop in this case, and no info is exposed.

Also it looks like this mutation does not touch any other system (e.g. Master Data), only the user sessions like @polishq  said, so I don't think we would be open to any kind of attack.

I'm fine with leaving it open to unauthenticated users.